subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheIntegrationTest.groovy

```diff
@@ -552,6 +553,7 @@ class ConfigurationCacheIntegrationTest extends AbstractConfigurationCacheIntegr
         type                             | reference                                                   | invocation
         Logger.name                      | "logger"                                                    | "info('hi')"
         ObjectFactory.name               | "objects"                                                   | "newInstance(SomeBean)"
+        ResourceHandler.name             | "resources"                                                 | "toString()"
```

Comment:
❌  Since `ResourceHandler` is not a service, it would be better to have it in a new test named something other than _restores task fields whose value is service of type #type_.

Reply:
It's the same for the existing test iteration for `Logger`. I'll move those two to a separate test.

Reply:
I see. Moving will make things less confusing, thanks!